packages/sign-client/test/sdk/push.spec.ts

```diff
@@ -5,29 +5,29 @@ import {
   testConnectMethod,
   deleteClients,
   TEST_EMIT_PARAMS,
-  TEST_RELAY_URL,
   TEST_WEBHOOK_ENDPOINT,
   throttle,
 } from "../shared";
+import { TEST_RELAY_URL } from "./../shared/values";
 import { describe, it, expect, afterEach, beforeEach } from "vitest";
 
-describe.skip("Push", () => {
+describe("Push", () => {
   let clients;
   let sessionA;
   beforeEach(async () => {
     clients = await initTwoClients();
+    console.log(
+      `Clients initialized (relay '${TEST_RELAY_URL}'), client ids: A:'${await clients.A.core.crypto.getClientId()}';B:'${await clients.B.core.crypto.getClientId()}'`,
+    );
     sessionA = (await testConnectMethod(clients)).sessionA;
   });
   it("receives a prompt webhook", async () => {
-    // Register Webhook for topic
-    await axios.post(
-      `${TEST_RELAY_URL.replace("ws", "http")}/subscribe`,
-      { webhook: TEST_WEBHOOK_ENDPOINT, topic: sessionA.topic },
-      {
-        headers: { "content-type": "application/json" },
-      },
+    await throttle(200); // Allow to propagate routing table
```

Comment:
Is this required on this test only or is nice to have on all after `testConnectMethod`?

Reply:
When I added this I didn't actually require it. But hypothetically there is a delay so might as well.